cypress/e2e/item/create/createFolder.cy.ts

```diff
@@ -33,11 +38,19 @@ describe('Create Folder', () => {
 
       // create
       createFolder(CREATED_ITEM);
+    });
 
-      cy.wait('@postItem').then(() => {
-        // expect update
-        cy.wait('@getItem').its('response.url').should('contain', id);
-      });
+    it('create folder with blank name in item', () => {
```

Comment:
```suggestion
    it('cannot create folder with blank name in item', () => {
```

Reply:
@pyphilia 
Thanks for your comment, Creating document with blank name is done